src/main/java/bio/terra/service/snapshot/SnapshotService.java

```diff
@@ -130,12 +130,18 @@ public String deleteSnapshot(UUID id, AuthenticatedUserRequest userReq) {
         .submit();
   }
 
-  public String exportSnapshot(UUID id, AuthenticatedUserRequest userReq, Boolean exportGsPaths) {
+  public String exportSnapshot(
+      UUID id,
+      AuthenticatedUserRequest userReq,
+      Boolean exportGsPaths,
```

Comment:
should these be primitive booleans (and a default for the api method) so we don't need to deal with nullability?

Reply:
I'm unsure of exactly how openApi handles defaults. Could someone specifically set `exportGsPaths=null` in their request?

Reply:
We can force it to be an explicit true or false.  But I believe that the the codegen adds a default value in the parameter annotation which spring then interprets by setting a default value